ush/parsing_namelists_FV3.sh

```diff
@@ -383,18 +406,9 @@ if [ ${DO_LAND_PERT:-"NO"} = "YES" ]; then
 EOF
 fi
 
-case ${gwd_opt:-"2"} in
-  1)
-  cat >> input.nml <<EOF
-  gwd_opt      = 1
-  do_ugwp      = .false.
-  do_ugwp_v0   = .false.
-  do_ugwp_v1   = .false.
-  do_tofd      = .true.
-  $gfs_physics_nml
-/
+if [ $knob_ugwp_version -eq 0 ]; then
```

Comment:
Why did you change the way this was done?

Reply:
Because the if statements no longer worked with the required settings for p8b.  @yangfanglin might be able to elaborate more.